### DISCUSSION TITLE
Not working in my laptop

### DISCUSSION THREAD

**Discussion by dev-keshav-raj** [Schneegans/Burn-My-Windows#227] [Q&A] [answered]
After installing the extensions it doesn't work .
when i open extensions manager it shows me this message

{
**GLib.FileError: Failed to open file “/home/*username/.local/share/gnome-shell/extensions/burn-my-windows@schneegans.github.com/resources/burn-my-windows.gresource”: open() failed: No such file or directory
}**

How i solve this now ?

**Comment by Schneegans** [comment, marked as answer]
Hi there! Well, if you installed the extension via one of the ways outlined in the [README](https://github.com/Schneegans/Burn-My-Windows#%EF%B8%8F-installation-gnome), this shouldn't happen.

* How did you install the extension?
* Which version of GNOME?
* Which operating system are you using?

**Reply by dev-keshav-raj** [reply]
Manual installation , by github .zip  file
OS: Ubuntu 22.04.1 LTS x86_64
Kernel: 6.0.0-060000-generic
DE: GNOME 42.4

**Reply by Schneegans** [reply]
Well, this zip file contains the file which is mentioned in the error message. Did you properly unzip the archive...?

**Reply by dev-keshav-raj** [reply]
yes i do all and i properly move it .

**Reply by Schneegans** [reply]
Well, something must have gone wrong during the installation. Does the file exist which is mentioned in the error? Could you please explain how you installed the extension in detail?

**Reply by dev-keshav-raj** [reply]
No, their is no any file or folder are exist , as per error , manual install from git hub zip and move in extension folder (manual added) another extension works fine and it also install by manual install .

**Reply by Schneegans** [reply]
Well, the file *is* in the zip. So there must have happened some kind of error. The installation process is very simple. It's just extracting a zip file - could you just do it again? And then check whether all files from the zip got properly extracted...?

**Reply by Schneegans** [reply]
Has this been solved on your end?

**Reply by dev-keshav-raj** [reply]
yes now its done ,Thank you
